### PR TITLE
Do not use sphinx action for building docs

### PR DESCRIPTION
**Pull request by maxnoe**
Trying to fix the docs build by not using the sphinx extension.

We loose the automatic PR comments for sphinx warnings though.

### CODE REVIEW

**Comment by maxnoe** [comment]
Ok, I think we should merge this to unblock the other PRs, we can switch back to the sphinx action if/when it gets updated.

**Review by kosack — APPROVED** [review]

**Review by LukasNickel — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -65,18 +65,21 @@ jobs:
...
+        uses: actions/setup-python@v2
         with:
-          docs-folder: "docs/"
-          pre-build-command: |
-            apt update --yes && apt install --yes git build-essential pandoc
-            pip install -U pip setuptools wheel
-            pip install -e .[docs]
-            git describe --tags
-            python -c 'import ctapipe; print(ctapipe.__version__)'
+          python-version: 3.8
```
Comment (HealthyPear) [inline .github/workflows/ci.yml:71]: why is it necessary to specify the python version here?

Comment (maxnoe) [reply .github/workflows/ci.yml:71]: Why should we rely on whatever is the current default?

Comment (HealthyPear) [reply .github/workflows/ci.yml:71]: I'm asking because just above you use `${{ matrix.python-version }}` which can be also 3.7

Comment (maxnoe) [reply .github/workflows/ci.yml:71]: This is for the unit tests, which we run for multiple python versions. It does not really make sense to build the documentation using multiple python versions.

Comment (HealthyPear) [reply .github/workflows/ci.yml:71]: exactly, so it should be "3.8" instead of "${{ matrix.python-version }}" (not a problem, though)

Comment (maxnoe) [reply .github/workflows/ci.yml:71]: I don't understand.

Comment (HealthyPear) [reply .github/workflows/ci.yml:71]: yes, but line 68 refers to the `docs` job, not to the `tests` one, so the name of the job should reflect the version we are using, which is only 3.8, that was my confusion (line 30 instead refers to the tests and is correct)

Comment (maxnoe) [reply .github/workflows/ci.yml:71]: Oh, there was a copy paste leftover above in the `name`. This is only for displaying, it does not influence the build.

Comment (HealthyPear) [reply .github/workflows/ci.yml:71]: yes I know, that's why I said it's not a problem, it just triggered a confusion from my side

**Pull request merged**